cypress.config.js

```diff
@@ -10,10 +10,17 @@ module.exports = defineConfig({
           const email = faker.internet.email();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + randomNumber,
+            username: faker.person.firstName() + randomNumber,
             email: email.toLowerCase(),
             password: '12345Qwert!'
           };
+        },
+        generateArticle() {
+          return {
+            title: faker.word.verb(),
+            description: faker.word.words(5),
+            body: faker.word.words(50)
```

Comment:
Consider using `faker.lorem.sentences(5)` instead of `faker.word.words(50)` for the body. This way, you ensure that the body is a coherent text, as `faker.word.words(50)` can return a list of unrelated words.